indigo/indigo/src/main/scala/indigo/platform/events/WorldEvents.scala

```diff
@@ -62,11 +65,15 @@ object WorldEvents {
       globalEventStream.pushGlobalEvent(
         MouseEvent.MouseUp(
           absoluteCoordsX(e.clientX) / magnification,
-          absoluteCoordsY(e.clientY) / magnification
+          absoluteCoordsY(e.clientY) / magnification,
+          MouseButton.fromOrdinal(e.button)
         )
       )
     }
 
+    // Prevent right mouse button from popping up the context menu
+    canvas.oncontextmenu = _.preventDefault()
```

Comment:
I think that's fair in general. The alternative would be for this to be the default, but to add a flag in the `AdvancedConfig` so the game dev can switch it on and off. I don't feel strongly about it though - you?

Reply:
Can we do this in another PR? 🤔 I think it makes sense to let the user decide about it, perhaps there are use cases I can not think of right now, i.e. something like integrating a game with the rest of a web page... I'm totally making up the use case though.